lira_scrum/models/project_agile_team.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2017 - 2018 Modoolar <info@modoolar.com>
+# License LGPLv3.0 or later (https://www.gnu.org/licenses/lgpl-3.0.en.html).
+
+from odoo import models, fields, api, tools, _
+
+
+class ScrumTeam(models.Model):
```

Comment:
this model and fields is already defined here: https://github.com/OCA/project-agile/pull/5/files#diff-58e8963716c00985642bc580a247c3d6R7

Repeated?